osmaxx-py/osmaxx/excerptexport/services/conversion_api_client.py

```diff
@@ -181,6 +182,21 @@ def update_order_status(self, extraction_order):
         else:
             return False
 
+    def estimated_file_size(self, north=0, west=0, south=0, east=0):
```

Comment:
Yes, keyword arguments and defaults can be used separately. In fact, every argument can be used as a keyword argument at the caller's discretion, whether it has a default or not.